src/goto-analyzer/static_analyzer.cpp

```diff
@@ -124,57 +109,62 @@ void static_analyzert::plain_text_report()
     {
       if(!i_it->is_assert()) continue;
 
-      tvt r=eval(i_it);
+      exprt simplified=domain[i_it].domain_simplify(i_it->guard, ns);
 
       result() << '[' << i_it->source_location.get_property_id()
                << ']' << ' ';
 
       result() << i_it->source_location;
+
       if(!i_it->source_location.get_comment().empty())
         result() << ", " << i_it->source_location.get_comment();
+
       result() << ": ";
-      if(r.is_true())
-        result() << "SUCCESS";
-      else if(r.is_false())
-        result() << "FAILURE";
-      else
-        result() << "UNKNOWN";
-      result() << eom;
 
-      if(r.is_true())
-        pass++;
-      else if(r.is_false())
-        fail++;
+      if(simplified.is_true())
+      {
+        result() << "SUCCESS"; pass++;
+      }
+      else if(simplified.is_false())
+      {
+        result() << "FAILURE (if reachable)"; fail++;
+      }
       else
-        unknown++;
+      {
+        result() << "UNKNOWN"; unknown++;
+      }
+
+      result() << eom;
     }
 
     status() << '\n';
   }
 
-  status() << "SUMMARY: " << pass << " pass, " << fail << " fail, "
+  status() << "SUMMARY: " << pass << " pass, " << fail << " fail if reachable, "
```

Comment:
If this is intended as user-visible status info, then it shouldn't be all-caps (see above).